### PR TITLE
Update: Add logging to yavin-client

### PR DESCRIPTION
**Pull request by kevinhinterlong**
## Description
Adds some basic logging to the yavin client

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/client/src/plugins/elide/adapters/facts.ts [hunk 1]
```diff
@@ -65,6 +66,8 @@ type PaginationOptions = {
   after: number;
 };
 
+const PLUGIN_NAMESPACE = 'plugins:elide:facts:adapter';
```
Comment (jkusa) [inline packages/client/src/plugins/elide/adapters/facts.ts:69]: Should we have `yavin` somewhere in the namespace?

```suggestion
const PLUGIN_NAMESPACE = 'yavin:plugins:elide:facts:adapter';
```
??

File: packages/client/src/services/fact.ts [hunk 2]
```diff
@@ -53,19 +58,19 @@ export default class FactsService extends NativeWithCreate implements FactServic
...
   }
 
   fetch(request: RequestV2, options: RequestOptions = {}): Promise<NaviFactsModel> {
-    return run(withLabels(this.fetchTask(request, options), { name: 'fetch' }));
+    return run(this.fetchTask(request, options), { labels: { name: 'services:facts:fetch' } });
   }
 
   fetchNext(response: NaviFactResponse, request: RequestV2): Promise<NaviFactsModel | null> {
-    return run(withLabels(this.fetchNextTask(response, request), { name: 'fetchNext' }), {});
+    return run(this.fetchNextTask(response, request), { labels: { name: 'fservices:facts:etchNext' } });
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/client/src/services/fact.ts:69]: maybe put `yavin:services:facts:fetchNext`

File: packages/client/src/plugins/elide/adapters/facts.ts (hunk 1 above)
Comment (kevinhinterlong) [reply packages/client/src/plugins/elide/adapters/facts.ts:69]: https://github.com/yavin-dev/framework/blob/c6cfe6e75ec3f3dcbbbfe9074ca00a1a62232c9f/packages/client/src/models/native-with-create.ts#L70
These extend from the main debug logger (default to yavin:client), so it's still in the name

File: packages/client/src/index.ts [hunk 3]
```diff
@@ -58,6 +62,7 @@ export class Client {
       dimensions: (_injector: Injector) => {
         throw new Error('Dimension service does not exist yet');
       },
+      logger: (_injector: Injector) => debug('yavin:client'),
```
Comment (jkusa) [inline packages/client/src/index.ts:65]: Can we consolidate this string with the one in packages/data/addon/services/yavin-client.ts

**Review by cythrawll — APPROVED** [review]

**Review by jkusa — APPROVED** [review]

**Pull request merged**